Source/Libraries/Adapters/GrafanaAdapters/GrafanaDataSourceBase_AncillaryOperations.cs

```diff
@@ -362,8 +383,20 @@ private static Task ProcessDistribution<T>(List<DataSourceValueGroup<T>> queryVa
                     GeoCoordinate newCoordinate = coordinateReference.Translate(translation, zoom);
                     map["Longitude"] = $"{newCoordinate.Longitude}";
                     map["Latitude"] = $"{newCoordinate.Latitude}";
+                    if (!string.IsNullOrEmpty(groupBy))
+                    groupMap.Add(FunctionParsing.ParseLabel(groupBy, metadata), new Point(newCoordinate.Longitude,newCoordinate.Latitude));
```

Comment:
Add four spaces before `groupMap...` for proper code alignment, plus add spacing before `newCoordinate.Latitude` parameter `...new Point(newCoordinate.Longitude, newCoordinate.Latitude)...`